### ISSUE TITLE
Floating an Image prevents spaces from being inserted

### ISSUE THREAD

**Issue by blairzotron** [tinymce/tinymce#5351]
Once I add an image and float it right, spaces can no longer be added (e.g. spacebar does absolutely nothing). The text and image are both inside the same <p> and the image has no unusual styles, just style="float: left;" 

![Spacebar-Issue](https://user-images.githubusercontent.com/5322175/72535815-8f8a5180-384f-11ea-9258-287b2342396f.gif)

**Comment by lostkeys** [comment]
Weird! Unfortunately I'm unable to replicate your issue. Are you comfortable sharing your TinyMCE config and your content css? Or try to [replicate it using our Fiddle](http://fiddle.tinymce.com)? Thanks.

**Comment by blairzotron** [comment]
Not sure if I created the Fiddle properly, but I was able to reproduce the issue:
http://fiddle.tinymce.com/PZgaab

**Comment by blairzotron** [comment]
Yeah, sorry, that's not a helpful Fiddle, because you have to type something, add an image, then float it, then place the cursor at the end of the text and start typing at which point spaces stop working.  Just as you see in my GIF.

**Comment by lostkeys** [comment]
I was able to replicate it with chrome, see http://fiddle.tinymce.com/RZgaab. Safari and Firefox works fine. It's a bug, thanks for finding it!

**Comment by blairzotron** [comment]
**Unrelated comment (sorry):**  in 4.x you could do **_stateSelector: 'a[href]'_** in editor.ui.register.addButton to make it highlight the button when, in this case, a link is selected; how do you do this in 5.x??   https://stackoverflow.com/questions/59847443/tinymce-5-x-highlight-custom-button
